skinfo/metrics.py

```diff
@@ -302,8 +353,73 @@ def kullback_leibler(x, y, bins):
         D_{KL}(x||y) = H(x||y) - H(x)
 
     """
-    # calculte the cross entropy and unconditioned entopy of y
-    hcross = cross_entropy(x, y, bins)
-    hx = entropy(x, bins)
+    # get the bins, only calculate if bins were not calculated before ([array, array])
+    if type(bins) == list:
+        if len(bins) == 2:
+            if (type(bins[0]), type(bins[1])) == (np.ndarray, np.ndarray):
+                pass
```

Comment:
I don't get this. And also please use `isinstance(bins, list)` and maybe even check for list and tuple: `isinstance(bins, (list, tuple))`.